engine/src/main/java/com/datatorrent/stram/plan/logical/LogicalPlanConfiguration.java

```diff
@@ -836,11 +1539,11 @@ public LogicalPlanConfiguration addFromJson(JSONObject json, Configuration conf)
     JSONArray operatorArray = json.getJSONArray("operators");
```

Comment:
I agree. But this is pre-existing code not directly related to my change. Would it be acceptable to file another ticket and do that work separately?